tests/test_rooms/test_create_room.py

```diff
@@ -54,6 +56,26 @@ def test_room_creation_with_duplicate_name_in_the_same_location(self):
             room_mutation_query_duplicate_name_response
         )
 
+    def test_invalid_room_label_format(self):
+        """
+        Test that room label does not accepts list of
+        dictionaries and saves them as strings
+        """
+        CommonTestCases.admin_token_assert_in(
+            self,
+            invalid_room_label_query,
+            "Room label is not a valid string type")
+
+    def test_nonexistent_structure_room_label_query(self):
+        """
+        Test that a room label cannot be created when
+        structure does not exist
+        """
+        CommonTestCases.admin_token_assert_in(
+            self,
+            non_existent_structure_room_label_query,
+            "Structure does not exist")
+
```

Comment:
@Arusey Kindly add test to show that `supplying a list of strings as room label` when creating a room will result in successful creation.

Reply:
Done